### ISSUE TITLE
Can't display TextArea and TextField

### ISSUE THREAD

**Issue by r3d9u11** [haxeui/haxeui-xwt#3]
## Current Behavior
Instread of windows with TextArea and TextField I see an empty windows.
![err1](https://cloud.githubusercontent.com/assets/15020827/23396828/b5dec172-fda6-11e6-8218-b9fe8c7306a3.png)

## Test app / minimal test case
UI.xml of [TextArea sample](https://github.com/r3d9u11/haxe-basics/tree/master/9_AdditionalLibraries/9.4_GuiFrameworks/9.4.2_HaxeUI/9.4.2.3_haxeui-xwt/9.4.2.3.1_Components/TextArea):
```
<?xml version="1.0" encoding="utf-8" ?>
<textarea wrap="false" width="200" height="200"/>
```
UI.xml of [TextField sample](https://github.com/r3d9u11/haxe-basics/tree/master/9_AdditionalLibraries/9.4_GuiFrameworks/9.4.2_HaxeUI/9.4.2.3_haxeui-xwt/9.4.2.3.1_Components/TextField):
```
<?xml version="1.0" encoding="utf-8" ?>
<textfield width="100" text="hello world..."/>
```

## Your Environment
Haxe 3.4.0
Linux Mint 18.1 x64
hxcs 3.4.0
haxeui-core from git
haxeui-xwt from git

Thanks.

**Comment by ianharrigan** [comment]
i could be wrong (@ibilon?) but pretty sure these components are implemented yet in haxeui-xwt.

**Comment by r3d9u11** [comment]
@ianharrigan I'm not sure, but I think that [TextEntry](https://github.com/mono/xwt/blob/master/Xwt/Xwt/TextEntry.cs) it's ananlog of TextField, for example.

**Comment by ianharrigan** [comment]
Yup, but its not linked / mapped here: https://github.com/haxeui/haxeui-xwt/blob/master/haxe/ui/backend/native.xml

**Comment by ibilon** [comment]
Yes this issue is the unfinished native.xml, I'll complete it soon.
